docs/_guide/templates.md

```diff
@@ -569,6 +581,162 @@ html`${until(content, html`<span>Loading...</span>`)}`
 
 For a list of directives supplied with lit-html, see [Built-in directives](https://lit-html.polymer-project.org/guide/template-reference#built-in-directives) in the Template syntax reference.
 
+## Accessing nodes in the shadow DOM
+
+Templated DOM is usually rendered into shadow DOM, so the nodes are not direct children of the component. Use `this.shadowRoot.querySelector()` or `this.shadowRoot.querySelectorAll()` to find nodes in the 
+shadow DOM.
+
+You can query the templated DOM after its initial render (for example, in `firstUpdated`), or use a getter pattern, like this:
+
+```js
+get closeButton() {
+  return this.shadowRoot.querySelector('#close-button');
+}
+```
+
+LitElement supplies a set of decorators that provide a shorthand way of defining getters like this.
+
+More information:
+
+*   [Element.querySelector()](https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelector) on MDN.
+*   [Element.querySelectorAll()](https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelectorAll) on MDN.
+
+### @query, @queryAll, and @queryAsync decorators
+
+The `@query`, `@queryAll`, and `@queryAsync` decorators all provide a convenient way to access nodes in the component's shadow root. 
+
+<div class="alert alert-info">
+
+**Using decorators.** Decorators are a proposed JavaScript feature, so you’ll need to use a compiler like Babel or the TypeScript compiler to use decorators. See [Using decorators](decorators) for details.
+
+</div>
+
+The `@query` decorator modifies a class property, turning it into a getter that returns a node from the render root.
+
+```js
+class MyElement {
+  @query('#first')
```

Comment:
I would suggest making this a TypeScript example, making the field private, and naming it with a `_` prefix, because the queried elements are internal implementation details and `@query` is really meant to internal usage - we don't want authors accidentally making private things public.

Reply:
At the moment, we don't have anything to show it's a Typescript example, so I've been trying to limit Typescript-specific syntax to examples that are explicitly labeled Typescript (one more reason I'm dying to get a working code-switcher up and running).

How about I just use the `_` and add a note about how these properties should typically be protected or private (`_` convention or Typescript's `private` or `protected` keywords).